docs/root/start/sandboxes/cache.rst

```diff
@@ -0,0 +1,289 @@
+
+Cache Filter
+============
+.. TODO(yosrym93): When a documentation is written for a production-ready Cache Filter, link to it through this doc.
+
+In this example, we demonstrate how HTTP caching can be utilized in Envoy by using the Cache Filter. 
+The setup of this sandbox is based on the setup of the :ref:`Front Proxy sandbox <install_sandboxes_front_proxy>`.
+
+All incoming requests are routed via the front Envoy, which is acting as a reverse proxy sitting on
+the edge of the ``envoymesh`` network. Ports ``8000`` and ``8001`` are exposed by docker
+compose (see :repo:`/examples/cache/docker-compose.yaml`) to handle ``HTTP`` calls
+to the services, and requests to ``/admin`` respectively. Two backend services are deployed behind the front Envoy, each with a sidecar Envoy.
+
+The front Envoy is configured to run the Cache Filter, which will store cacheable responses in an in-memory cache, 
+and serve it to subsequent requests. In this demo, the responses that are served by the deployed services are stored in :repo:`/examples/cache/responses.yaml`. 
+This file is mounted to both services containers, so any changes made to the stored responses while the services are running should be instantly effective (no need to rebuild or rerun). 
+For the purposes of the demo, a response's date of creation is appended to its body before being served. 
+An Etag is computed for every response for validation purposes, which only depends on the response body in the yaml file (i.e the appended date is not taken into account). 
+Cached responses can be identified by having an ``age`` header. Validated responses can be identified by having a generation date older than the ``date`` header;
+as when a response is validated the ``date`` header is updated, while the body stays the same.
+
+Running the Sandbox
+~~~~~~~~~~~~~~~~~~~
+
+The following documentation runs through the setup of an Envoy cluster organized
+as is described in the image above.
+
+**Step 1: Install Docker**
+
+Ensure that you have a recent versions of ``docker`` and ``docker-compose`` installed.
+
+A simple way to achieve this is via the `Docker Desktop <https://www.docker.com/products/docker-desktop>`_.
+
+**Step 2: Clone the Envoy repo**
+
+If you have not cloned the Envoy repo, clone it with:
+
+``git clone git@github.com:envoyproxy/envoy``
+
+or
+
+``git clone https://github.com/envoyproxy/envoy.git``
+
+**Step 3: Start all of our containers**
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/cache
+    $ docker-compose build --pull
+    $ docker-compose up -d
+    $ docker-compose ps
+
+           Name                      Command               State                             Ports                          
+    ------------------------------------------------------------------------------------------------------------------------
+    cache_front-envoy_1   /docker-entrypoint.sh /bin ...   Up      10000/tcp, 0.0.0.0:8000->8000/tcp, 0.0.0.0:8001->8001/tcp
```

Comment:
i think if you have more than one space here it fails formatting - ie after the `...` - see here https://github.com/envoyproxy/envoy/pull/12401/commits/25d87ca0d1c98ba97d4923ebaced643a113c473d

Reply:
wow - fixed quicker than i could type!

Reply:
You started reviewing while I was still actively on the PR! Fast reviews = fast fixes!